docs/tutorials/hackathon-prep-guide/7-obtaining-cycles.md

```diff
@@ -0,0 +1,184 @@
+
+Click on the `REQUEST CYCLES` button on the faucet web page to join the Discord server.
+
+![Getting Coupon](_attachments/faucet_step_1.png)
+
+Once inside the Discord server, navigate into the `#cycles-faucet` channel. 
+
+![Cycles-faucet](_attachments/cycles-faucet.png)
+
+In this channel, execute the following slash command:
+
+```
+/request
+```
+
+This command will call the IC Cycles Faucet Bot. This bot will send you a survey that will need to be filled out and submitted. Once the survey has been completed, our team will review your submission. If accepted, the faucet bot will send you a private message with a coupon code.
+
+:::info
+Please ensure that your Discord settings are set to allow direct messages from other users. If you do not have this setting enabled, you will not receive a direct message from the faucet bot.
+:::
+
+Once you have your coupon code, head back to the <https://faucet.dfinity.org> webpage. 
+
+Now, click **NEXT STEP** to continue. Once you have a coupon code, enter your coupon code within the faucet UI.
+
+![Enter Coupon](_attachments/faucet_step_3.png)
+
+Click **NEXT STEP** to continue.
+
+Return to your terminal window. If you closed the window in the process, run the following commands to start `dfx` and use your previously created identity:
+
+```sh
+dfx start --background
+dfx identity use MyIdentity
+```
+
+Then, redeem your coupon with the command, replacing **COUPON_CODE** with the cycles coupon code sent to you by the DFINITY faucet bot:
+
+```sh
+dfx wallet --network ic redeem-faucet-coupon COUPON_CODE
+```
+
+Verify your cycles balance with the command:
+
+```sh
+dfx wallet --network ic balance
+```
+
+This should return an output showing 10T cycles in your wallet. To get the canister identifier of your new cycles wallet, run the command:
+
+```sh
+dfx identity --network ic get-wallet
+```
+
+## Converting ICP tokens to cycles
+
+If you've already redeemed a cycles coupon in the past, you can convert ICP tokens into cycles. Before you can convert ICP tokens into cycles, first you need to obtain ICP tokens. ICP tokens can be purchased through a crypto exchange, or they can be received through other activities such as participating in the NNS governance and receiving grants from the DFINITY foundation.
+
+To get your account ID so you know where to send your ICP tokens, run the command:
+
+```
+dfx ledger account-id
+```
+
+Once you have sent some ICP to this account ID, you can verify that they were received by checking the balance with the command:
+
+```
+dfx ledger --network ic balance
+```
+
+After you have your ICP tokens ready, you will need to create a cycles wallet if you haven't already. This will require your identity principal, which you can obtain by running the command:
```

Comment:
we should include instructions to check if the current identity already has a cycles wallet on mainnet